sdk/data/aztables/client.go

```diff
@@ -653,7 +658,7 @@ func (s *SetAccessPolicyOptions) toGenerated() *generated.TableClientSetAccessPo
 // Specify nil for options if you want to use the default options.
 func (t *Client) SetAccessPolicy(ctx context.Context, options *SetAccessPolicyOptions) (SetAccessPolicyResponse, error) {
```

Comment:
While fixing the panic, I wonder if `SetAccessPolicyOptions` should actually be optional?  Would one ever call this API with a `nil` options?

Reply:
To clarify, should `TableACL` be optional?

Reply:
That's a good point. TableACL probably should not be optional. I'm surprised this didn't come up in archboard. I wonder if by specifying nil you can remove access policies. Let me investigate.

Reply:
I've removed the other changes so this PR is just doc updates now.